R/recipe_run.R

```diff
@@ -741,7 +741,7 @@ recipe_copy_readme <- function(info, src) {
 
     ## now check if README is a resource
     if (length(info$resources) > 0) {
-      i <- grepl("README.md", info$resources, ignore.case = FALSE)
```

Comment:
consider above doing

```
  readme_files <- dir(src, pattern = "README(\\.md)?$",
                      ignore.case = TRUE, recursive = TRUE)
```

then adapting the if to be `if (length(readme_files) > 1) {...`

then copying *all* of the readme files over - that would require some variant of `dir_create(file.path(dest, dirname(readme_files))` before the copy and a mapping onto the canonical filename (`sub("README(.\\md)$". "README\\1", readme_files)`) before doing the copy

Reply:
Should the logic be '*copy across all READMEs we find*'?

This came up because there was report that listed a directory as a resource that contained a readme and there was no way to get rid of the unexpected artefact warning.

Do we want to support the use case where a report directory contains a file that is not copied across?